openslides/core/static/templates/index.html

```diff
@@ -39,7 +39,9 @@
     <div class="user" ng-controller="userMenu">
       <div class="btn-group">
         <div ng-if="operator.isAuthenticated()">
-
```

Comment:
Why does git recognize this line in fact i doesn't changed. Or does it? (please check your line endings)